configuration/editors.md

```diff
@@ -15,6 +15,17 @@ This documentation page can give you some guidance in choosing the right one for
 - TOC
 {:toc}
 
+### Network Preparations
+
+The editors need to be able to access the configuration files on the remote openHAB host.
+
+You have to have a [network share](https://en.wikipedia.org/wiki/Shared_resource) set up on the remote host and mounted on your local computer.
+The required steps to do so are specific to the host operation system.
```

Comment:
Consider, "The steps required to set up a `network share` on your local host computer are specific to..."